pages/linux/apache2ctl.md

```diff
@@ -0,0 +1,24 @@
+# apache2ctl
+
+> The CLI tool apache2ctl (alternatively apachectl)is a front end to the Apache HyperText Transfer Protocol (HTTP) server.  It is designed to help the administrator control the functioning of the Apache apache2 daemon.
```

Comment:
This can be shortened. I would write it something like `CLI to administer the apache2 HTTP daemon.`

Reply:
Thanks Agniva for suggestions. Let me resolve these and send PR again.